src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -1717,3 +1706,30 @@ void PosixSignals::do_resume(OSThread* osthread) {
 
   guarantee(osthread->sr.is_running(), "Must be running!");
 }
+
+void os::SuspendedThreadTask::internal_do_task() {
+  if (PosixSignals::do_suspend(_thread->osthread())) {
+    os::SuspendedThreadTaskContext context(_thread, _thread->osthread()->ucontext());
+    do_task(context);
+    PosixSignals::do_resume(_thread->osthread());
+  }
+}
+
+int PosixSignals::init() {
+  // initialize suspend/resume support - must do this before signal_sets_init()
+  if (SR_initialize() != 0) {
+    vm_exit_during_initialization(err_msg("SR_initialize failed"));
```

Comment:
err_msg not needed, just use the plain literal

Reply:
Ah, `err_msg()` is for the cases where we need to print out values (i.e. formatting).